### ISSUE TITLE
Input for give_level_start_date displayed as plain text for Give all non-members level ID section

### ISSUE THREAD

**Issue by ipokkel** [strangerstudios/pmpro-toolkit#14]
The Give all non-members level ID section displays the start date input box as plain text

https://github.com/strangerstudios/pmpro-toolkit/blob/86467040644d60eceb4dfb8639f6579f97d03a1b/adminpages/scripts.php#L368

![firefox_gn00k07qyt](https://user-images.githubusercontent.com/9744519/79144849-f2907380-7dbf-11ea-8a5e-7bc79e92eb1e.png)

As a temporary workaround this line can be replaced with this:

```PHP
<?php echo esc_html_e( 'Give all non-members level ID ', 'pmpro-toolkit' ); ?><input type="text" name="give_level_id" value="" size="4" />. <?php echo esc_html_e( 'Set the start date to ', 'pmpro-toolkit' ) ?><input type="text" name="give_level_startdate" value="" size="10" /> <?php echo esc_html_e( '(YYYY-MM-DD) and set the end date to ', 'pmpro-toolkit' ); ?><input type="text" name="give_level_enddate" value="" size="10" /> <?php echo esc_html_e( '(optional, YYYY-MM-DD)', 'pmpro-toolkit' ) ?>.
```

**Comment by ipokkel** [comment]
A PR with a similar solution is availabe here:https://github.com/strangerstudios/pmpro-toolkit/pull/9/files

**Comment by dparker1005** [comment]
Merged #18, which has a fix for this.
